devp2p/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use devp2p::cli::DummyConfig;
+use devp2p::dummy_protocol::DummyProtocol;
+use log::info;
+use trin_core::portalnet::Enr;
+use trin_core::portalnet::U256;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tracing_subscriber::fmt::init();
+
+    tokio::spawn(async move {
+        let (p2p, events) = DummyProtocol::new().await.unwrap();
+
+        tokio::spawn(events.process_discv5_requests());
+
+        let dummy_config = DummyConfig::new();
+
+        let target_enrs: Vec<Enr> = dummy_config
+            .target_nodes
+            .iter()
+            .map(|nodestr| nodestr.parse().unwrap())
+            .collect();
+
+        // Test Ping target nodes
+        for enr in &target_enrs {
+            info!("Pinging {} on portal network", enr);
+            let ping_result = p2p
+                .send_ping(U256::from(u64::MAX), enr.clone())
+                .await
+                .unwrap();
+            info!("Portal network Ping result: {:?}", ping_result);
```

Comment:
Presumably the idea is to either panicking here or keep track of failures to display them at the end of the run. Is that your plan for a follow-up PR?

Reply:
I'm currently exploring different possibilities, this code is just to make sure it is working in the first place :)